libs/schedulers/include/hpx/schedulers/queue_holder_thread.hpp

```diff
@@ -134,8 +134,7 @@ namespace hpx { namespace threads { namespace policies {
 
         static util::internal_allocator<threads::thread_data> thread_alloc_;
 
-        typedef util::tuple<thread_init_data, thread_state_enum>
-            task_description;
+        typedef util::tuple<thread_init_data> task_description;
```

Comment:
Does this still have to be a `tuple` now?

Reply:
No. I've changed other instances, but missed this one. Thanks!